lib/plugins/login/login.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const crypto = require('crypto');
+const readline = require('readline');
+const opn = require('opn');
+const fetch = require('node-fetch');
+const jwtDecode = require('jwt-decode');
+const chalk = require('chalk');
+const getFrameworkId = require('../../utils/getFrameworkId');
+const updateConfig = require('../../utils/config/update');
+
+const config = {
```

Comment:
hmmm I'm super curious about that. Would anyone be able to just copy that into their code and implement their own login? Do we want people logged in outside of the context of the framework?